doc/conf.py

```diff
@@ -170,15 +167,6 @@
 #modindex_common_prefix = []
 
 nitpicky = True
-nitpick_ignore = [ ('cpp:typeOrConcept', 'std')
```

Comment:
Here I'd much rather have
```
nitpick_ignore = [
                 ]
```
in case this is needed in the future.

Reply:
Will add